src/claimtrie.cpp

```diff
@@ -620,9 +620,17 @@ bool CClaimTrie::getClaimById(const uint160 claimId, std::string& name, CClaimVa
     CClaimIndexElement element;
     if (db.Read(std::make_pair(CLAIM_BY_ID, claimId), element))
```

Comment:
So the claimId passed in here isn't always utilized/enforced?

Reply:
Not sure what you're asking.

Reply:
I was wondering why we have to verify the returned claimId (just below this) when we requested a specific one here.

Reply:
We don't.  I added logging in case of corruption (as in current code).  That shouldn't ever happen.